mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java

```diff
@@ -46,7 +46,7 @@
 
 
 	/**
-	 * Returns a list of length N atoms of the B-factors of the atoms as Integers.
+	 * Returns an array of length N atoms of the B-factors of the atoms as integers.
 	 * They must be divided by 100.0 to be in float form.
```

Comment:
Yes both good points.

On Fri, Mar 25, 2016 at 4:24 PM Jose Manuel Duarte <notifications@github.com>
wrote:

> In mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java
> <https://github.com/rcsb/mmtf-java/pull/3#discussion_r57501984>:
>
> > @@ -46,7 +46,7 @@
> >
> >
> >  	/**
> > -	 * Returns a list of length N atoms of the B-factors of the atoms as Integers.
> > +	 * Returns an array of length N atoms of the B-factors of the atoms as integers.
> >  	 * They must be divided by 100.0 to be in float form.
>
> We'd also need a constant for the 100.0
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rcsb/mmtf-java/pull/3/files/a839340374ccc73595e675af7ed33e1208b13213#r57501984>
>